lib/temporal/workflow/history/event_target.rb

```diff
@@ -16,6 +16,7 @@ class UnexpectedEventType < InternalError; end
         CANCEL_EXTERNAL_WORKFLOW_REQUEST_TYPE = :cancel_external_workflow_request
         WORKFLOW_TYPE                         = :workflow
         CANCEL_WORKFLOW_REQUEST_TYPE          = :cancel_workflow_request
+        SIGNAL_EXTERNAL_WORKFLOW_TYPE         = :external_workflow
```

Comment:
You can probably reuse the already existing `EXTERNAL_WORKFLOW_TYPE` here

Reply:
Ah, that was a mistake. Removed.